### DISCUSSION TITLE
voicenotify during phone calls

### DISCUSSION THREAD

**Discussion by Skall36** [pilot51/voicenotify#112] [Q&A] [answered]
Hi, would it be possible to have voicenotify working during phone calls or would it cause audio problems?

**Comment by pilot51** [comment, marked as answer]
It is currently hard coded to stay silent during phone calls.

When I originally implemented it in 2011 (commit d50235a), I didn't think anyone would want it to speak during calls as it would distract the conversation and likely be heard through the call (a potential privacy issue). I've received a number of complaints about it speaking during calls, such as issue #111. I believe you are the first to ask about letting it speak during calls.

I'm not against adding the option, though I would consider it low priority given the lack of demand.
Would you like me to convert your question into a feature request?

**Reply by Skall36** [reply]
Thank you for your reply and explanation. i understand that my request is specific:
I ride a onewheel, and during my rides, I'm regularly on the phone with a friend. the onewheel application sends notifications if there's a problem, and recently I had a rather violent fall due to insufficient battery power to support my speed. I thought that I might have been able to avoid it if I had heard the notification, so I looked for an application that allowed this and installed yours, which seemed to be suitable, but I couldn't figure how to make it work during phone calls.
Having said that, I could have avoided the fall in many other ways, such as not being distracted by the phone call or going slower.

Don't bother making a request about this issue as I must be the only one in the world to have it.

Thank you again for listening so carefully to your users.

Regards

**Reply by pilot51** [reply]
You make a strong case for giving the option and I can imagine other people having similar safety-related scenarios. I know a lot of people ride onewheels and many would pass the time with calls, podcasts, or music.

It would make more sense once #87 is done, so you could configure important apps to speak during calls without allowing it for all the other apps you normally have enabled.
